src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -267,8 +267,7 @@ private string BuildUnexpectedErrorMessage(string methodName, Exception exceptio
 
         private static async Task<object> ExecuteHubMethod(ObjectMethodExecutor methodExecutor, THub hub, object[] arguments)
         {
-            // ReadableChannel is awaitable but we don't want to await it.
-            if (methodExecutor.IsMethodAsync && !IsChannel(methodExecutor.MethodReturnType, out _))
```

Comment:
I couldn't find any reference of `ReadableChannel` and no tests failed when I removed this...

Reply:
They must have removed that feature from Channels.

Reply:
It's [`ChannelReader`](https://github.com/dotnet/corefx/blob/master/src/System.Threading.Channels/src/System/Threading/Channels/ChannelReader.cs) now and they did indeed remove the awaitable feature.